pitch.md

```diff
@@ -1,44 +1,160 @@
-# Project Title (replace me)
+# ComplexCalendar
 
 ## Overview
 _What's this project about, briefly?_
 
+This project is about making complex calendar's easy to make. For any calendar
+software it is easy to make specific events but for most people, their life
+heavily revolves around a core calendar that repeats and doesn't change often.
+I want to make it easier for users to define their complex calendars in fewer
+words. 
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+The user for this project would be someone that has a highly structured calendar
+that is too complex to easily define with a simple repeat item button. An example
+of one user would be a college student. Their calendar for classes only applies 
+when school is in session. It would be much easier for the student to define a 
+school section of their calendar and mark the dates of the school year to apply
+it. This would eliminate their need to go months forward in the calendar to set 
+up new repeated events that were already set up for other times. 
+
+While a UI would probably be the final product of this project, it would be 
+nicely represented with a backend language. An expressive language allows
+for small changes to have large effects. If a user's calendar is always 
+recorded in the language instead of chunked time-slots, making a small change
+to the initial design of the calendar would just require a small change in the
+language, not having to go event by event to make that small change. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need that is met by my idea is the need for complex calendar design. Current
```

Comment:
While this is true, I think you could be more explicit that this is really only intended for related and repeating events - the extra work of a one-off event is probably not worth the hassle of writing a program.